src/parse/rules/cc_rules.build_defs

```diff
@@ -822,34 +850,43 @@ def _binary_cmds(c, linker_flags, pkg_config_libs, extra_flags='', shared=False,
                   CONFIG.CC_TOOL if c else CONFIG.CPP_TOOL]
 
 
-def _library_transitive_labels(c, compiler_flags, pkg_config_libs, archive=True):
+def _library_transitive_labels(c, compiler_flags, pkg_config_libs, pkg_config_cflags, archive=True):
     """Applies commands from transitive labels to a cc_library rule."""
     def apply_transitive_labels(name):
         labels = get_labels(name, 'cc:')
         flags = ['-isystem %s' % l[4:] for l in labels if l.startswith('inc:')]
         flags.extend(['-D' + l[4:] for l in labels if l.startswith('def:')])
-        pkg_config_libs.extend([l[3:] for l in labels if l.startswith('pc:') and l[3:] not in pkg_config_libs])
+
+        if pkg_config_libs:
```

Comment:
I don't think these should be conditional, presumably it should extend `pkg_config_cflags` in the same way as the logic worked before